exe/server/Dockerfile

```diff
@@ -1,11 +1,21 @@
 FROM golang:1.13.7 as builder
```

Comment:
Make the Dockerfile of _fil-tools_ better. 
Instal _rustc_ for Linux and not use the copied compiled things from the host (in the `COPY . .` line).

Realized about this problem when I changed to macOS, since the compiled rust git-submodule was for darwin arch, and the container is Linux. This will make a completely clean build of fil-tools in the docker image

Reply:
So glad you switched to Mac hahah. Although maybe the benefit comes from switching back and forth every once in a while :)